templates/cli/lib/config.js.twig

```diff
@@ -408,59 +415,140 @@ class Global extends Config {
         super(`${homeDir}/${path}`);
     }
 
+    getCurrentLogin() {
+        if (!this.has(Global.PREFERENCE_CURRENT)) {
+            return "";
+        }
+        return this.get(Global.PREFERENCE_CURRENT);
+    }
+
+    setCurrentLogin(endpoint) {
+        this.set(Global.PREFERENCE_CURRENT, endpoint);
+    }
+
+    getLoginIds() {
+        return Object.keys(this.data).filter((key) => !Global.IGNORE_ATTRIBUTES.includes(key));
+    }
+
+    getLogins() {
+        const logins = Object.keys(this.data).filter((key) => !Global.IGNORE_ATTRIBUTES.includes(key))
+
+        return logins.map((login) => {
+
+            return {
+                id: login,
+                endpoint: this.data[login][Global.PREFERENCE_ENDPOINT],
+                email: this.data[login][Global.PREFERENCE_EMAIL]
+            }
+        })
+    }
+
+    addLogin(login, data) {
+        this.set(login, data);
+    }
+
+    removeLogin(login, data) {
+        this.delete(login);
+    }
```

Comment:
Lets refactor all these method names to use Session / Session 

And deleteSession / logout